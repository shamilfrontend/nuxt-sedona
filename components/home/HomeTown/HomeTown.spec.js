import { shallowMount } from '@vue/test-utils'
import Component from './index'

describe('HomeTown', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Component, {})
  })

  test('mounts properly', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
