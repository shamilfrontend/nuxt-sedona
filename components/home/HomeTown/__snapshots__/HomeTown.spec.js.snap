// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeTown should match snapshot 1`] = `
<section class="home-town">
  <div class="home-town__content">
    <h3 class="home-town__title">Настоящий городок</h3>
    <p class="home-town__number">— №1 —</p>
    <p class="home-town__text">
      Седона — не аттракцион для туристов, там течёт своя жизнь
    </p>
  </div>
  <div class="home-town__image"></div>
</section>
`;
